### DISCUSSION TITLE
Miller Indices visualizer :Lattice Plane

### DISCUSSION THREAD

**Discussion by Freya36910** [mtex-toolbox/mtex#1839] [Ask Anything]
Hello,
I need a code to draw orientation of a crystal plane in a lattice is specified by Miller Indices. Could you please help me in this regard.
Freya

**Comment by kilir** [comment]
Hi,
`plot(Miller(h,k,l,cs))` should usually do.
Cheers,
Rüdiger

**Comment by Freya36910** [comment]
Thanks for prompt response.
I run "Plotting crystal shapes" from MTEX but it gives me just shape, I am wondering if there is any code for crystal shape with slip plane on it.
Farzaneh

**Comment by Freya36910** [comment]
![Miller](https://github.com/mtex-toolbox/mtex/assets/134095325/51e36537-cebb-4602-8e95-29ba68c4df29)

**Comment by kilir** [comment]
Hi
now I see. Something like that? 
You need to adapt it to your needs.
```
cS = crystalShape.cube(crystalSymmetry('432'))

nS = cS; 
nS.V=nS.V([3 5 8])
nS.F=[1 2 3]

plot(cS,'FaceAlpha',0.3)
hold on
plot(nS,'FaceColor','r','FaceAlpha',0.3)
hold off
```

**Reply by Freya36910** [reply]
Many Thanks Rüdiger,
I am working on Aluminum (FCC). Should I change just 'FaceAlpha' to 'Aluminum'
Freya

**Reply by kilir** [reply]
No, of  course not.
Cheers,
Rüdiger

**Reply by Freya36910** [reply]
I do appreciate it.
Best Regards,
Freya
